### DISCUSSION TITLE
Cara Mengajukan Pertanyaan

### DISCUSSION THREAD

**Discussion by rioastamal** [kirimwa-id/forum#2] [Tanya Jawab]
Halo Semua,

Kategori ini dikhusukan untuk tanya-jawab seputar penggunaan API KirimWA.id. Supaya pertanyaan kamu dapat dijawab dengan efektif usahakan pertanyaan yang dibuat lebih detil. Pastikan kamu sudah membaca [dokumentasi API](https://developer.kirimwa.id/api/v1/) sebelum mengajukan pertanyaan karena mungkin jawaban pertanyaan yang akan kamu tanyakan sudah ada disana.

> **Penting!** Jangan mencantumkan informasi yang bersifat rahasia seperti API Token, password dan sebagainya.

### Contoh Salah

> Saya tidak bisa mengirim pesan! Tolong dibantu.

Ini pertanyaan yang terlalu umum dan tidak spesifik. Sehingga tidak bisa dijawab langsung karena banyak hal yang belum jelas. Seperti parameter yang digunakan dan respon yang didapat.

### Contoh Benar

> Saya mendapat error ketika mengirim pesan gambar.

```
{
  "message": "Failed to send message. Error: Invalid image URL."
}
```
> Apa yang diharapkan?

```
{
  "id": "kwid-426564a5db7940288dc9fddb845",
  "status": "pending",
  "message": "Message is pending and waiting to be processed.",
  "meta": {
    "location": "https://api.kirimwa.id/v1/messages/kwid-426564a5db7940288dc9fddb845"
  }
}
```

> Cara menduplikasi atau perintah yang dijalankan.

```bash
curl -X POST 'https://api.kirimwa.id/v1/messages' \
-H 'Authorization: Bearer TOKEN_SAYA` \
-H 'Content -Type: application/json` \
-d '{
  "phone_number": "628123456789",
  "message": "Halo ini gambar https://example.com/gambar.jpg",
  "device_id": "iphone-x",
  "message_type": "image"
}
```

**Comment by asep2004ganjar** [comment]
Maff om, kok token sya invalid token

**Reply by rioastamal** [reply]
Silahkan buat thread baru dengan detil perintah apa saja yang telah dilakukan, seperti contoh diatas. Sembunyikan token (ganti dengan ***) ketika melakukan copy-paste ke forum.

**Comment by asep2004ganjar** [comment]
![git](https://user-images.githubusercontent.com/24951568/210161002-92eaa69f-348e-44ed-9615-23867c9a4903.PNG)

Token saya invalid ,
Perintah yang saya pake :

saya hanya mengujinya dengan perintah sbb

hasilnya

{"message":"Invalid API token."}

maaf , saya baru di git hub

**Comment by asep2004ganjar** [comment]
udah bisa om maaf,balas disini

**Comment by asep2004ganjar** [comment]
udah bisa om , udah bisakirim 
terimakasih banyak

**Comment by asep2004ganjar** [comment]
Lapor Om, sudah bisa nangkap data g dikirim via webhook om.  
seperti ini om data nya om
![kirimwa](https://user-images.githubusercontent.com/24951568/210223709-a0da0df2-b722-4a2c-ab83-0c54490bb29e.PNG)

**Comment by hs6778367-commits** [comment]
cara membuka blokir kartu ATM BNI Menghubungi:+628137072274-))aplikasi seluler bernama 'wondr'. Teks dalam bahasa Indonesia menjelaskan bahwa proses ini dapat dilakukan tanpa perlu mengantri di layanan pelanggan atau mesin. Gambar menampilkan antarmuka aplikasi 'wondr' pada smartphone, menyoroti kemudahan dan kenyamanan dalam menggunakan aplikasi untuk layanan

**Comment by hs6778367-commits** [comment]
Ya Untuk layanan buka Blokir BNI mobile banking tanpa ke bank Atau bisa meng hubungi WA ((-08217555452-))layanan Call Center untuk ...

**Comment by hs6778367-commits** [comment]
cara membuka blokir kartu ATM BNI Menghubungi:+628137072274-))aplikasi seluler bernama 'wondr'. Teks dalam bahasa Indonesia menjelaskan bahwa proses ini dapat dilakukan tanpa perlu mengantri di layanan pelanggan atau mesin. Gambar menampilkan antarmuka aplikasi 'wondr'

**Comment by hs6778367-commits** [comment]
Ya Untuk layanan Call Center UOB kini hadir 24jam tanpa ke bank Atau ke cabang hubungi WA 08137422+8047 layanan ini bisa di hubungi untuk masalah transaksi bermasalah dan informasi lebih lanjut silahkan untuk mengikut
